pkg/kapp/config/default.go

```diff
@@ -48,6 +48,12 @@ rebaseRules:
   resourceMatchers:
   - allMatcher: {}
 
+- path: [metadata, annotations, "kapp.k14s.io/last-renewed-time"]
+  type: copy
+  sources: [new, existing]
```

Comment:
This is not set by user but kapp internally will update this value when condition matches. If will do `sources: [existing]` then it will always copy the old value (which is in exiting res) which we do not want.